content/nginx-one-console/nginx-configs/certificates/csg-unmanaged-certificates.md

```diff
@@ -0,0 +1,115 @@
+- Certificate consistency is determined by comparing certificate contents and file paths across instances
+
+### Consistent certificates
+
+When all instances in a CSG reference identical certificate files with the same file paths:
+
+- Their contents and metadata match across all instances
+- The CSG displays a single unmanaged certificate entry for that file path
+
+### Inconsistent certificates
+
+If certificate contents differ between instances, even when file paths are the same:
+
+- Each unique certificate appears as a separate unmanaged entry in the Console
+- Certificates are identified by their content and associated instance
+- The CSG displays separate certificate entries in the configuration
+
+If certificate file paths differ between instances:
+
+- CSG publication may fail
+- The CSG configuration will be out of sync
+- Instances may not receive proper configuration updates
+
+## Requirements for unmanaged certificates
+
+To use unmanaged certificates effectively in Config Sync Groups, you must:
+
+- **Manual installation**: Install certificates manually on each NGINX instance in the CSG
+- **Identical file paths**: Ensure that file paths referencing unmanaged certificates are identical across all instances
+- **Content consistency**: Maintain identical certificate file contents across all instances to ensure proper tracking
+- **User responsibility**: Take full responsibility for certificate distribution, updates, and consistency
+
+## Important considerations
+
+### Certificate tracking
+
+- The NGINX One Console tracks unmanaged certificates by their content and file paths
+- When certificates are consistent across all instances, their contents and metadata match, and a single consolidated entry appears in the CSG
+- If certificate content differs between instances, multiple unique unmanaged certificates are displayed as separate entries
+
+### Synchronization limitations
+
+- **No automated sync**: Unmanaged certificates are not synchronized by the Console
+- **Manual updates**: You must manually update certificates on each instance when they expire or need rotation
+- **No validation**: The Console does not perform validation or rotation logic for unmanaged certificates
+
+### Configuration options
+
+If you don't want metadata tracking for unmanaged certificates, you can configure the NGINX Agent to ignore certificate directories using the `allowed_directories` setting.
+
+## Best practices
+
+### Converting to managed certificates
+
+To maintain consistent visibility and automated management across CSGs, consider converting unmanaged certificates to managed certificates by:
+
+1. Uploading them through the NGINX One Console
+2. Leveraging the managed certificate solution for automated synchronization
+3. Taking advantage of centralized certificate management features
+
+## Troubleshooting
+
+### Certificate inconsistencies
+
+If you see multiple entries for what should be the same certificate:
```

Comment:
I see you've numbered these options. In general, I number steps when users have to do them, in order. 

If ordering is not required, I'd replace the numbers with bullets